hw/ip/clkmgr/doc/_index.md

```diff
@@ -231,12 +231,12 @@ There is currently no support in hardware to dynamically synchronize a threshold
 
 The table below summarises the valid modes and the settings required.
 
-| Mode                         | `lc_ctrl_clk_byp_req`  | `extclk_ctrl.sel` | `extclk_ctrl.hi_speed_sel`  | life cycle state        |
-| -------------                | ---------------------  | ----------------- | ----------------------------| ----------------------- |
-| Life cycle transition        | `lc_ctrl_pkg::On`      | Don't care        | Don't care                  | Controlled by `lc_ctrl` |
-| Internal Clocks              | `lc_ctrl_pkg::Off`     | `kMultiBit4False` | Don't care                  | All                     |
-| Software external high speed | `lc_ctrl_pkg::Off`     | `kMultiBit4True`  | `kMultiBit4True`            | TEST_UNLOCKED, RMA      |
-| Software external low speed  | `lc_ctrl_pkg::Off`     | `kMultiBit4True`  | `kMultiBit4False`           | TEST_UNLOCKED, RMA      |
+| Mode                                            | `lc_clk_byp_req_i`     | `extclk_ctrl.sel` | `extclk_ctrl.hi_speed_sel`  | life cycle state        |
+| -------------                                   | ---------------------  | ----------------- | ----------------------------| ----------------------- |
+| Life cycle transition from RAW and TEST* states | `lc_ctrl_pkg::On`      | `kMultiBit4False` | Don't care                  | Controlled by `lc_ctrl` |
```

Comment:
I see, you are probably right: if lc_clk_byp_req_i and sw requests external clock things can become confusing. Does this mean we cannot do lc_ctrl transitions once external clock is selected? Could that prevent TEST* transitions in some cases? For example, if the clocks are unstable and we use SW extclk to bypass the oscillators we would not be able to run LC transitions?

Reply:
ah sorry @matutem, let me add you to the thread, it will be clearer.

Reply:
We can leave aside the weird case I am concerned about.